addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
because i the addon should know the device already. it is passed in the props. create is really a start() call now.